### PR TITLE
Implement task management feature

### PR DESCRIPTION
**Pull request by ruppfabian1997**
Issue: #107

### CODE REVIEW

**Comment by ruppfabian1997** [comment]
This Pull Request can be set to "ready to review"

File: src/layout/AppTopbar.vue
Comment (FelixSiegfriedRiedel) [inline src/layout/AppTopbar.vue:1]: `onAllTasksClick()` is not fully implemented yet but It is nice that the function got added already. 
When implementing it, it needs to be assured that the route exists too.

**Review by FelixSiegfriedRiedel — COMMENTED** [review]
In general a very good implementation for the tasks management that acts as a good foundation for future revisions. The layout could be slightly improved and some redundancies eg. name/title and unused functions could be removed. 
More tests could be added too. 
The general structure is great and there is even decent error handling present.
Thank you!

File: src/services/TaskService.ts [hunk 1]
```diff
@@ -0,0 +1,101 @@
+import axios from "axios";
+
+export interface Task {
+    id: string;
+    title: string;
+    description: string;
+    status: Status;
+    ownerId: string;
+    created_at: Date;
+    modified_at: Date;
```
[the remark this answers is not in the document]
Comment (ruppfabian1997) [reply src/services/TaskService.ts:10]: Good Point. i changed it to lowerCamelCase to be more consistent!

File: src/layout/AppTopbar.vue [hunk 2]
```diff
@@ -39,6 +39,11 @@ const onNewProjectClick = () => {
   router.push('/new-project');
 };
 
+const onAllTasksClick = () => {
+  topbarMenuActive.value = false;
+  router.push({name: "AllTasksOverview", params: {projectId: projectStore.projectId}});
+};
+
 const onAccountSettingsClick = () => {
```
[the remark this answers is not in the document]
Comment (ruppfabian1997) [reply src/layout/AppTopbar.vue:47]: You're right. We only need the Button in the ManagerMenu. I deleted it from the Topbar Menu.

File: src/services/TaskService.ts [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+}
+
+export interface TaskList {
+    tasks: TaskItem[];
+}
+
+export interface TaskItem {
+    id: string;
+    name: string;
+    title: string
```
[the remark this answers is not in the document]
Comment (ruppfabian1997) [reply src/services/TaskService.ts:31]: The name and title are taken from the backend. However, the reasons should be discussed!

File: src/layout/AppTopbar.vue
Comment (ruppfabian1997) [reply src/layout/AppTopbar.vue:1]: Yes, that's right. The implementation of the entire task view is not yet complete and should be further developed in new issues

**Comment by ruppfabian1997** [comment]
I resolved the merge Issues and passed the Quality Gate on sonarcloud! 😄

**Pull request merged**